src/lucky_cli/task.cr

```diff
@@ -7,6 +7,19 @@ abstract class LuckyCli::Task
     def name
       "{{@type.name.gsub(/::/, ".").underscore}}"
     end
+
+    def help_message
+      "Run this task with `lucky #{name}`"
+    end
+
+    def print_help_or_call(args : Array(String))
+      if args.any? { |arg| ["--help", "-h", "help"].includes?(arg) }
```

Comment:
I think it would be good to do something like this `args.size == 1 && %w(-h --help help).includes(args.first)`. That way it only works with the first arg. Also might be worth extracting a method like `wants_help_message?(args)` or something like that

Reply:
This one is a little tricky because technically if `args.first` is `--help`, then it prints the main help with `lucky --help`. So it might have to be something like `args.size == 2` or `args.size > 1` and `[].includes? args[1]`

Reply:
Ohhh I see. Let's stick with what you've got then. I have some ideas for making this whole thing more robust in the future. I did it differently in `up` and it is a lot nicer. Basically duplicates the args and removes the first arg once it finds a matching command